### PR TITLE
Release notes logging

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by mburke5678** [comment]
@sichvoge PTAL

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -552,7 +591,7 @@ In the table below, features are marked with the following statuses:
 
 |Log forwarding
 |TP
-|GA
+|TP
```
Comment (mburke5678) [inline release_notes/ocp-4-6-release-notes.adoc:594]: @sichvoge As i understand, we are not backporting the Log Forwarding GA to 4.5?

**Comment by sichvoge** [comment]
LGTM 👍

**Comment by mburke5678** [comment]
Closing in favor of: https://github.com/openshift/openshift-docs/pull/26353

**Pull request closed without merging**